test/e2e/host_path.go

```diff
@@ -28,36 +27,25 @@ import (
 	client "k8s.io/kubernetes/pkg/client/unversioned"
 
 	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
 )
 
 //TODO : Consolidate this code with the code for emptyDir.
 //This will require some smart.
 var _ = Describe("hostPath", func() {
-	var (
-		c         *client.Client
-		namespace *api.Namespace
-	)
+	framework := Framework{BaseName: "hostpath"}
```

Comment:
Done. It was written this way pretty much in every test (until @smarterclayton fixed it yesterday).